internal/api/attestationconfigapi/client.go

```diff
@@ -91,50 +90,33 @@ func (a Client) deleteAzureSEVSNPVersion(versions AzureSEVSNPVersionList, versio
 		},
 	})
 
-	ops = append(ops, deleteCmd{
-		apiObject: AzureSEVSNPVersionSignature{
-			Version: versionStr,
-		},
-	})
-
 	removedVersions, err := removeVersion(versions, versionStr)
 	if err != nil {
 		return nil, err
 	}
 	ops = append(ops, putCmd{
 		apiObject: removedVersions,
+		signer:    a.signer,
 	})
 	return ops, nil
 }
 
-func (a Client) uploadAzureSEVSNP(versions AzureSEVSNPVersion, versionNames []string, date time.Time) (res []crudCmd, err error) {
+func (a Client) uploadAzureSEVSNP(versions AzureSEVSNPVersion, versionNames []string, date time.Time) []crudCmd {
```

Comment:
This function should be named something like `build...` or `construct...`, as there is no upload happening in the function. Imo it should also end in a noun that makes sense (`...version`, `...request`), "SEVSNP" isn't a thing to do something with.